homeassistant/components/evohome/water_heater.py

```diff
@@ -112,6 +112,14 @@ async def async_turn_away_mode_off(self):
         """Turn away mode off."""
         await self._evo_broker.call_client_api(self._evo_device.set_dhw_auto())
 
+    async def async_turn_on(self):
+        """Turn on."""
+        await self._evo_broker.call_client_api(self._evo_device.set_dhw_on())
+
+    async def async_turn_off(self):
+        """Turn off."""
+        await self._evo_broker.call_client_api(self._evo_device.set_dhw_off())
```

Comment:
Should these have tests?

Reply:
I am not sure what you mean:
 - there is no data passed into `async_turn_on()`, `async_turn_off()`- so there's nothing to validate
 - the whole integration has no QA testing, so I don't think appropriate to add that now